test/polyfill/media_capabilities_unit.js

```diff
@@ -163,7 +163,7 @@ describe('MediaCapabilities', () => {
           expect(result.keySystemAccess).toEqual(mockResult);
         });
 
-    it('should read previously requested codec/key system'+
+    it('should read previously requested codec/key system ' +
```

Comment:
Why?

Reply:
To add a missing space between `system` and `combinations`. Burned my eyes when I saw this test failing.